src/clj/game/cards/hardware.clj

```diff
@@ -394,6 +394,18 @@
                 {:msg "remove 1 tag" :label "[Trash]: Remove 1 tag"
                  :effect (effect (trash card {:cause :ability-cost}) (lose :tag 1))}]}
 
+   "Friday Chip"
+    (let [ability {:msg (msg "move 1 virus counter to " (:title target))
+                   :req (req (pos? (get-in card [:counter :virus] 0)))
+                   :choices {:req #(and (has-subtype? % "Virus")
+                                        (pos? (get-in % [:counter :virus] 0)))}
```

Comment:
I think this is out of place -- Friday Chip does not require the target to have a virus counter already. We should also enforce "virus _program_"... so far only programs are viruses, but you never know.